examples/compiled/stacked_bar_count_corner_radius_config.vg.json

```diff
@@ -48,7 +48,7 @@
         "facet": {
           "data": "source_0",
           "name": "stack_group_main",
-          "groupby": ["month_date"],
+          "groupby": ["month_date", "month_date_end"],
```

Comment:
For this particular spec, `month_date_end` is not being used, but I guess it doesn't hurt to add it there?

Reply:
Yeah, probably doesn't matter. We do generate this as an output from time unit earlier anyway.